content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,12 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Fix Mixer unable to start when Galley is disabled ([Issue 14841](https://github.com/istio/istio/issues/14841))
+- Add HTTP/1.0 support to gateway configuration generation ([Issue 13085](https://github.com/istio/istio/issues/13085))
+- Redis filter (this feature is guarded with the environment feature flag PILOT_ENABLE_REDIS_FILTER, disabled by default):
```

Comment:
Needs to actually say something beyond "Redis filter", and sentence ends with a colon.